test/functional/fixtures/api/es-next/assertions/test.js

```diff
@@ -1,4 +1,8 @@
 const expect = require('chai').expect;
+const fs = require('fs');
+const path = require('path');
+
+const dataPath = './data';
```

Comment:
1. We use upper-snake case for constants:
```
const DATA_PATH = "..."
```
2. Let's construct the absolute path right here.